CHANGELOG.md

```diff
@@ -67,6 +67,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [SIL.Windows.Forms] Made FadingMessageWindow implement all UI logic on the main UI thread in a thread-safe way. Fixes crashes like SP-2340.
 - [SIL.Windows.Forms] Made ContributorsListControl more threadsafe. Possibly fixes crashes like SP-2353 or at least makes them less likely.
 - [SIL.WritingSystems] Added check to Subtag.Equals to ensure two subtags being compared are of same derived type. This could potentially be a subtle breaking change in the unlikely event that someone was intentionally relying on the previous errant behavior.
+- [SIL.Core.ClearShare] BREAKING CHANGE: Public method LoadWorkFromXml could possibly return a contribution with null role; in those cases, now falls back to the first role in the list of roles in the doc.
```

Comment:
I'm not very sure this is desirable. If a contribution was added but no role was specified, I don't think we can safely default the role.

Reply:
Perhaps not. And if nothing else, that decision is outside the main scope of this pr.